lib/network/kmip/index.ts

```diff
@@ -28,6 +28,23 @@ function _PrimitiveType(tagName: string, type: string, value: any) {
     return { [tagName]: { type, value } };
 }
 
+/** Thales has a 24h idle tls session error. We can close and reopen new connection */
+const thalesInternalTokenError = {
+    status: 'Operation Failed',
+    reason: 'General Failure',
+    messages: [
+        '[NCERRUnauthorizedAccess]: Token has been revoked',
+        '[NCERRUnauthorizedAccess]: Invalid token'
+    ],
+};
+
+//Could test on the kmip conf providerName as well, Should test other providers too
```

Comment:
```suggestion
// Could test on the kmip conf providerName as well, should test other providers too
```

Reply:
whats the goal of this comment?

Reply:
This is a thales specific behavior that we discovered in prod when some clients had some problems.

But we also support hashicorp or hytrust for kmip providers and I have no idea if they have such a thing as well. So it was more like a todo but I can drop the comment from the code

Reply:
If its todo, a ticket should be linked to this.